direct/train.py

```diff
@@ -75,11 +75,13 @@ def get_root_of_file(filename: PathOrString):
 
 
 def build_transforms_from_environment(env, dataset_config: DictConfig) -> Callable:
+    masking = dataset_config.transforms.masking  # Masking func can be None
+    mask_func = None if masking is None else build_masking_function(**masking)
```

Comment:
Perhaps some logging would be useful here, I can imagine a small typo would lead to masking = None, a log is helpful in auditing